<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HyperlinkUi</class>
 <widget class="QWidget" name="HyperlinkUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="targetLA">
         <property name="toolTip">
          <string>URL</string>
         </property>
         <property name="text">
          <string>&amp;Target:</string>
         </property>
         <property name="buddy">
          <cstring>targetED</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nameLA">
         <property name="toolTip">
          <string>Name associated with the URL</string>
         </property>
         <property name="text">
          <string>&amp;Name:</string>
         </property>
         <property name="buddy">
          <cstring>nameED</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="targetED">
         <property name="toolTip">
          <string>URL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameED">
         <property name="toolTip">
          <string>Name associated with the URL</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="literalCB">
           <property name="toolTip">
            <string>Pass content of the `Name' field literally to LaTeX. Check this if you want to enter LaTeX code.</string>
           </property>
           <property name="text">
            <string>Li&amp;teral</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="typeGB">
     <property name="toolTip">
      <string>Specify the link target</string>
     </property>
     <property name="title">
      <string>Link type</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="webRB">
        <property name="toolTip">
         <string>Link to the web or to every other target</string>
        </property>
        <property name="text">
         <string>&amp;Web</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="emailRB">
        <property name="toolTip">
         <string>Link to an email address</string>
        </property>
        <property name="text">
         <string>E&amp;mail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="fileRB">
        <property name="toolTip">
         <string>Link to a file</string>
        </property>
        <property name="text">
         <string>Fi&amp;le</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>112</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>targetED</tabstop>
  <tabstop>nameED</tabstop>
  <tabstop>webRB</tabstop>
  <tabstop>emailRB</tabstop>
  <tabstop>fileRB</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
